ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot

```diff
@@ -0,0 +1,167 @@
+    Edit Accelerator Profile    original_display_name=${ACC4DISPLAY_NAME}    enabled=False
+    Accelerator Profile Should Be Displayed In The Grid   ${ACC4_DISPLAY_NAME}
+    Verify Accelerator Profile Values Via CLI   ${ACC4_DISPLAY_NAME}    enabled=False
+
+Enable an Accelerator Profile From The Accelerator Profile Edit View
+    [Documentation]    Enable an An accelerator profile from the Edit Accelerator Profile view
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Run Keywords    Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}5
+    ...    AND
+    ...    Open Dashboard And Navigate to Accelerator Profiles From Settings
+    Edit Accelerator Profile    original_display_name=${ACC5_DISPLAY_NAME}    enabled=True
+    Accelerator Profile Should Be Displayed In The Grid   ${ACC5_DISPLAY_NAME}
+    Verify Accelerator Profile Values Via CLI   ${ACC5_DISPLAY_NAME}    enabled=True
+
+Disable an Accelerator Profile From The Grid of the Accelerator Profile View
+    [Documentation]    Disable an An accelerator profile from the Edit Accelerator Profile view
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Run Keywords    Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}6
+    ...    AND
+    ...    Open Dashboard And Navigate to Accelerator Profiles From Settings
+    Search for accelerator profile in grid    ${ACC6_DISPLAY_NAME}
+    Disable Accelerator Profile    ${ACC_NAME}6
+    Verify Accelerator Profile Values Via CLI   ${ACC6_DISPLAY_NAME}    enabled=False
+
+Enable an Accelerator Profile From The Grid of the Accelerator Profile View
+    [Documentation]    Enable an An accelerator profile from the Accelerator Profile view
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Run Keywords    Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}7
+    ...    AND
+    ...    Open Dashboard And Navigate to Accelerator Profiles From Settings
+    Search for accelerator profile in grid    ${ACC7_DISPLAY_NAME}
+    Click On The Enable Switch From Accelerator Profile    ${ACC_NAME}7
+    Verify Accelerator Profile Values Via CLI   ${ACC7_DISPLAY_NAME}    enabled=True
+
+Delete an Accelerator Profile From The Grid of the Accelerator Profile View
+    [Documentation]    Delete an An accelerator profile from the Accelerator Profile view
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Run Keywords    Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}8
+    ...    AND
+    ...    Open Dashboard And Navigate to Accelerator Profiles From Settings
+    Delete Accelerator Profile    ${ACC8_DISPLAY_NAME}
+    Accelerator Profile Should Not Exist   ${ACC_NAME}8
+
+
+*** Keywords ***
+Teardown Settings Accelerator Profiles
+    [Documentation]    Sets the default values In User Management Settings
+    ...                and runs the RHOSi Teardown
+    Dashboard Settings Accelerator Profiles Test Teardown
```

Comment:
Shouldn't the test teardown be called when finishing each test too?

Reply:
In the teardown the test just delete the created resources (Accelerator profile instances in this case).
The way I've done it is using the same preffix for all the instances, so it witll do an "oc get accelratorprofiles ..." filter the ones with the preffix and  then delete 'em.
I think that it's better to do this way instead of deleting each resource after each test because:
1. It's centralized
2. If someone wants to add some new tests, and follow the name structure as in the older tests (which is easy to realize because all tests use the variable ACC_NAME), does not need to delete anything

Reply:
probably you don't need the `Dashboard Settings Accelerator Profiles Test Teardown` keyword, but you could just move `Delete All Accelerator Profiles Which Starts With ${ACC_NAME}` inside the Suite Teardown

Reply:
Done [ecbabe7](https://github.com/red-hat-data-services/ods-ci/pull/1209/commits/ecbabe7a2c2442c9ab91cc5cd637dc11eac4a05d)